lib/webpacker/dev_server.rb

```diff
@@ -0,0 +1,54 @@
+# Same convention as manifest/configuration.rb
+# Loads webpacker configuration from config/webpacker.yml
+
+class Webpacker::DevServer < Webpacker::Configuration
```

Comment:
I'm trying to figure out what role this class plays that's separate from the configuration class. Most of the methods simply delegate. Is this just to overlay the ENV lookups? I don't think that's enough justification for the additional indirection.